### PR TITLE
Remove live input limits, find better homes for live streaming FAQ content

### PR DESCRIPTION
**Pull request by irvinebroque**
- Removes all live streaming content from the FAQ
- Adds RTMP reconnection and billing content to main Stream Live page
- Moves simulcast limit to simulcasting section, clarify intro sentence
- Clarify DASH limitation to be more specific
- Remove duplicate content around recording last 7 days of content

### CODE REVIEW

File: content/stream/stream-live/start-stream-live.md [hunk 1]
```diff
@@ -111,13 +111,14 @@ curl -X DELETE \
...
+- Closed GOPs are required. This means that if there are any B frames in the video, they should always refer to frames within the same GOP. This setting is default in most encoding software such as OBS.
 - Stream Live only supports H.264 video and AAC audio codecs as inputs. This requirement does not apply to inputs that are relayed to Stream Connect outputs. Stream Live supports ADTS but does not presently support LATM.
 - Clients must be configured to reconnect when a disconnection occurs. Stream Live is designed to handle reconnection gracefully by continuing the live stream.
 
 ### Known limitations
 
-- Stream Live currently only supports HLS (HTTP Live Streaming), and recordings are only kept for the last seven days of the stream.
+- DASH playback is not currently supported
```
Comment (irvinebroque) [inline content/stream/stream-live/start-stream-live.md]:
```suggestion
- Playback of live streams using DASH is not currently supported
```

**Review by renandincer — APPROVED** [review]

File: content/stream/stream-live/start-stream-live.md [hunk 2]
```diff
@@ -111,13 +111,14 @@ curl -X DELETE \
 
 ### Requirements
 
-- You must set [GOP duration](https://en.wikipedia.org/wiki/Group_of_pictures) (keyframe interval) to be between 2 to 10 seconds. The default in most encoding software, including Open Broadcaster Software (OBS), is within this range. Setting a lower GOP duration will reduce latency for viewers, while also reducing encoding efficiency. Setting a higher GOP duration will improve encoding efficiency, while increasing latency for viewers. This is a tradeoff inherent to video encoding, and not a limitation of Cloudflare Stream.
-- Closed GOPs required. This means that if there are any B frames in the video, they should always refer to frames within the same GOP. This setting is default in most encoding software such as OBS.
+- Stream supports a maximum bitrate of 12000 Kbps.
```
Comment (renandincer) [inline content/stream/stream-live/start-stream-live.md]: is it worth clarifying this is ingest bitrate?

Comment (irvinebroque) [inline content/stream/stream-live/start-stream-live.md]:
```suggestion
- Stream supports a maximum ingest bitrate of 12000 Kbps.
```

Comment (irvinebroque) [reply content/stream/stream-live/start-stream-live.md]: https://github.com/cloudflare/cloudflare-docs/pull/6495/commits/0d9e30a62e75c10c28bab37645c00134397027f5

**Pull request merged**